pre_commit/languages/all.py

```diff
@@ -16,6 +16,7 @@
 from pre_commit.languages import perl
 from pre_commit.languages import pygrep
 from pre_commit.languages import python
+from pre_commit.languages import renv
```

Comment:
I think this should maybe just be called `r`?

Reply:
Thought about that also. The thing is that `language: conda` can both host R and Python hooks and `renv` can [manage python dependencies](https://rstudio.github.io/renv/articles/python.html), so I thought I go for a more specific name. On the other hand,  `language: python` uses `venv` (and is still called `python`, not `venv`). Since working with virtual environments is still not as common in R compared to Python and not every R user knows `renv`, using `r` as a name might also make it easier for people to find what they are looking for. So I am happy to switch to `r`, please let me know what you prefer.

Reply:
yeah I think `r` makes the most sense, `conda` is kinda the special case right now and I think if people google "pre-commit r" they'll be much more likely to hit this than if it were named `renv`

we can always add other names if new ways of installing isolated r environments happen in the future and/or shuffle them around (there used to be a `python_venv` language which was merged into `python` as `virtualenv` makes `venv`-style environments anyway nowadays)